tuple/keydef.c

```diff
@@ -652,7 +661,8 @@ luaopen_tuple_keydef(struct lua_State *L)
 
 	JSON_PATH_IS_SUPPORTED = json_path_is_supported();
 
-	/* Export C functions to Lua. */
+	/* Create the module table. */
+	lua_createtable(L, 0, 6);
```

Comment:
Would you like to replace const 6 with `sizeof(meta)-1`?

Reply:
`lengthof(meta) - 1`, but, sure, I'll do.

Reply:
Done.